parlai/tasks/convai2/agents.py

```diff
@@ -105,3 +105,26 @@ def create_agents(opt):
     else:
         # interactive task has no task agents (they are attached as user agents)
         return []
+
+
+class PersonaTopicifierTeacher(SelfOriginalTeacher):
+    """
+    Adds WoW topics to ConvAI2 data.
+    """
+
+    def __init__(self, opt, shared=None):
+        from parlai_internal.projects.blended_skill_talk.add_personas_topics import (
```

Comment:
you can't add internal stuff to public parlai!?

Reply:
i'm not sure this should be in this PR either

Reply:
Ugh, good catch. Yes, agreed - removed